docs/usage/bot-comparison.md

```diff
@@ -0,0 +1,130 @@
+| Scheduling                                | Yes: `daily`, `weekly`, `monthly`                                                                                                                                            | Yes: custom                                                                                              |
+| License                                   | [The Prosperity Public License 2.0.0](https://github.com/dependabot/dependabot-core/blob/main/LICENSE)                                                                       | [GNU Affero General Public License](https://github.com/renovatebot/renovate/blob/main/license)           |
+| Programming language of project           | Ruby                                                                                                                                                                         | TypeScript                                                                                               |
+
+## Hosted app
+
+This section explains the key differences when you're using the Mend Renovate app or the GitHub-native Dependabot.
+
+If you're going to self-host a bot, read the hosted app section first anyway.
+Then read the self-hosted section.
+
+### Dependency Dashboard
+
+One big feature of Renovate is the Dependency Dashboard, which is enabled by default.
+Read the [Key concepts, Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) page to learn more.
+
+Dependabot does not have a similar feature.
+
+### Grouped updates
+
+Renovate comes with community-provided groupings of dependencies, so it will group common dependencies out-of-the-box.
+
+With Dependabot you'll have to set your own [`groups`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups).
+
+### Upgrades common monorepo packages at once
+
+Renovate has a [`group:monorepos`](https://docs.renovatebot.com/presets-group/#groupmonorepos) preset, that upgrades common monorepo packages at once.
+
+Dependabot does not update common monorepo packages at once.
+
+### Supported platforms
+
+Platform means the Git-hosting site or program, for example GitHub, GitLab or Azure.
+
+Dependabot only works on GitHub.
+
+Renovate works on multiple platforms, including GitHub.
+Read the [list of Renovate platforms](https://docs.renovatebot.com/modules/platform/) to learn more.
+
+### Supported languages
+
+- [Renovate's supported managers](https://docs.renovatebot.com/modules/manager/)
+- [Dependabot's supported repositories and ecosystems](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#supported-repositories-and-ecosystems)
+
+### Customizability
+
+Both Renovate and Dependabot come with good default settings.
+
+Renovate gives you more control over _when_ and _how_ you'll get the updates.
```

Comment:
Might be worthwhile to spell out some specifics here. This kind of gives the impression that there are _no_ customizable options for this with Dependabot.

Reply:
Agreed, let's avoid generalizations and use (meaningful) specifics instead - or leave such comparisons out

Reply:
I'll leave it out, I don't know off-hand how to make a meaningful comparison, and I don't want to hold up this PR too long.